docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -62,3 +65,92 @@ Diffie-Hellman key exchange
         :type: int
 
         The generator value.
+
+
+Key interfaces
+~~~~~~~~~~~~~~
+
+.. class:: DHParameters
+
+    .. versionadded:: 0.8
+
+
+.. class:: DHParametersWithSerialization
+
+    .. versionadded:: 0.8
+
+    Inherits from :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParameters`.
+
+    .. method:: parameter_numbers()
+
+        Return the numbers that make up this set of parameters.
+
+        :return: A :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParameterNumbers`.
+
+
+.. class:: DHPrivateKey
+
+    .. versionadded:: 0.8
+
+    .. attribute:: key_size()
+
+        The bit length of the prime modulus.
+
+    .. method:: public_key()
+
+        Return the public key associated with this private key.
+
+        :return: A :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHPublicKey`.
+
+    .. method:: parameters()
+
+        Return the parameters associated with this private key.
+
+        :return: A :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParameters`.
+
+    .. method:: exchange()
+
+        :returns: An instance of a
+            :class:`~cryptography.hazmat.primitives.asymmetric.KeyExchangeContext`
```

Comment:
It is a thing. Key exchange algorithm was removed though. I'm going to remove it from this PR though since the context PR isn't actually merged yet.